internal/dcache/cluster_manager/cluster_manager_impl.go

```diff
@@ -228,61 +259,182 @@ func (cmi *ClusterManagerImpl) stop() error {
 	if cmi.clusterMapticker != nil {
 		cmi.clusterMapticker.Stop()
 	}
+	if cmi.updatesChan != nil {
+		close(cmi.updatesChan)
+	}
 	return nil
 }
 
 // getActiveMVs implements ClusterManager.
-func (c *ClusterManagerImpl) getActiveMVs() []dcache.MirroredVolume {
-	return make([]dcache.MirroredVolume, 0)
+func (cmi *ClusterManagerImpl) getActiveMVs() map[string]dcache.MirroredVolume {
+	if cmi.localMap == nil {
+		log.Debug("ClusterManagerImpl::getActiveMVs: local cluster map not loaded")
```

Comment:
this shouldn't happen and should be asserted.
simply do

``
common.Assert(cmi.localMap != nil)
```